src/fmu/ensemble/realization.py

```diff
@@ -7,30 +7,30 @@
 an archived realization.
 """
 
-import os
-import re
 import copy
 import glob
 import json
-from datetime import datetime, date, time
-import dateutil
 import logging
+import os
+import re
 import warnings
+from datetime import date, datetime, time
 
-import yaml
+import dateutil
 import numpy as np
 import pandas as pd
+import yaml
 
-from resdata.summary import Summary
-from resdata.resfile import ResdataFile
-from resdata.rd_util import FileMode
```

Comment:
Is this ruff doing import sorting? I don't think ruff is sorting imports in CI, that would be nice to have in a separate commit before this.